stac_fastapi/extensions/stac_fastapi/extensions/third_party/bulk_transactions.py

```diff
@@ -12,13 +12,13 @@
 
 
 class Items(BaseModel):
-    """A list of STAC items."""
+    """A group of STAC Item objects, in the form of a dictionary from Item.id -> Item."""
 
     items: Dict[str, Any]
```

Comment:
I haven't followed all the discussion but do we need it as a dict? Or could we just have a list of stac items?

Reply:
a list of items would align better with `ItemCollection`

Reply:
Correct, I just wanted to fix the behavior rather than change the expected input